extensions/wikia/ThemeDesigner/SpecialThemeDesignerPreview.class.php

```diff
@@ -7,36 +7,33 @@ public function __construct() {
 	}
 
 	public function execute( $par ) {
-		wfProfileIn( __METHOD__ );
 		global $wgOut, $wgExtensionsPath;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)